templates/core/build.html

```diff
@@ -148,26 +148,13 @@ <h4 class="p-muted-heading">Pre-installed snaps</h4>
 
   <section class="p-strip--light is-shallow">
     <div class="row">
-      <h2>Summary</h2>
-    </div>
-    <div class="row p-divider">
-      <div class="col-6 p-divider__block">
-        <dl>
-          <dt>Board</dt>
-          <dd class="js-summary-board"></dd>
-          <dt>Operating system</dt>
-          <dd class="js-summary-os"></dd>
-          <dt>Packages</dt>
-          <dd><span class="js-summary-snaps">0</span> pre-installed packages</dd>
-        </dl>
-      </div>
-      <div class="col-6 p-divider__block">
-        <p>This will trigger a build in Launchpad to create your image. Once the image has been built we will contact you via email on <strong>{{ openid.email }}</strong>. This build will only be available for a few days so please do not rely on Launchpad for future downloads</p>
+      <div class="col-8">
         <p>
           <input class="mktoField" value="yes" id="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
           <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">I agree to receive information about Canonical’s products and services.</label>
         </p>
         <p><button class="p-button--positive js-build-button" aria-disabled="true" disabled="">Build image</button></p>
+        <p>We’ll mail you at {{ openid.email }} once the build is ready.</p>
```

Comment:
I don't like "We'll mail you". That sounds very archane language to me. I get that in this day and age email is so common that you would almost think it would be the default form of "mail"ing, but still, I don't think "mail" is a common term in modern parlance.

I'd suggest "We'll email you" or "We'll contact you".

I'm approving this anyway.

Reply:
@matthewpaulthomas what do you think?